keyboards/flx/virgo/rules.mk

```
@@ -16,8 +17,8 @@ BOOTLOADER = atmel-dfu
 BOOTMAGIC_ENABLE = lite      # Virtual DIP switch configuration
 MOUSEKEY_ENABLE = yes        # Mouse keys
 EXTRAKEY_ENABLE = yes        # Audio control and System control
-CONSOLE_ENABLE = yes         # Console for debug
-COMMAND_ENABLE = yes         # Commands for debug and configuration
+CONSOLE_ENABLE = no         # Console for debug
```

I'd like to see the comment alignment kept.

```suggestion
CONSOLE_ENABLE = no          # Console for debug
```